app/__init__.py

```diff
@@ -0,0 +1 @@
+default_app_config = 'app.apps.ShbAppConfig'
```

Comment:
Die Models müssen bei watson registriert werden - das wurde früher in der models.py gemacht, jetzt soll das in einer eigenen AppConfig passieren. Die Zeile gibt an, wo die zu verwendende AppConfig liegt.